raster/r.mapcalc/r3.mapcalc.html

```diff
@@ -466,19 +466,21 @@ <h3>3D GRID MASK handling</h3>
 r3.mapcalc "volume_const = 200.0"
 </pre></div>
 the created raster map is limited only by a computation region
-but it is not affected by an active MASK.
-This is expected because, as mentioned above, MASK is only applied when reading,
-not when writing a raster map.
-<p>
-If also in this case the MASK should be applied, an if() statement including the
-MASK should be used, e.g.:
+but it is not affected by an active raster mask.
+This is expected because, as mentioned above, the mask is only applied
+when reading, not when writing a raster map.
+If the raster mask should be applied in this case, an <code>if()</code>
+function including the mask raster should be used, e.g.
+(assuming the mask is called <code>MASK</code>):
 <div class="code"><pre>
 r3.mapcalc "volume_const = if(MASK, 200.0, null())"
 </pre></div>
 
-When testing MASK related expressions keep in mind that when MASK is active
+<p>
+When testing expressions related to mask handling keep in mind that
+when the raster mask is active,
 you don't see data in masked areas even if they are not NULL.
-See <em><a href="r.mask.html">r.mask</a></em> for details.
+See <em><a href="r3.mask.html">r3.mask</a></em> for details.
```

Comment:
r3.mask doesn't have many details, especially compared to r.mask that has examples, while r3.mask doesn't have some. r3.mask doesn't link to r.mask.

Reply:
Yes, well, as I said, the 3D text looks like a simple copy-paste from r.mapcalc. It is not that helpful anyway, but fixing that is out of scope here. r2.mask to r3.mask is an obvious simple change I can do within this PR.

Reply:
In other words, 3D raster documentation could use some love.

Reply:
> In other words, 3D raster documentation could use some love.

Ok with that, if r3.mask would be updated and has the information about the mask. Or replace to a canonical page that describes masks in details that can be used for multiple pages. For now it's only r.mask that has it (one place).